tools/c7n_azure/c7n_azure/query.py

```diff
@@ -27,6 +27,14 @@ class ResourceQuery(object):
     def __init__(self, session_factory):
         self.session_factory = session_factory
 
+    @staticmethod
+    def resolve(resource_type):
+        if not isinstance(resource_type, type):
+            raise ValueError(resource_type)
+        else:
```

Comment:
I copied/pasted from c7n/query. I’d probably leave it this way so they look the same. Agree?

Reply:
Makes sense

Reply:
either works, there is no gospel, direct return is pretty obvious here as well. the resolve type condition check and method here might even be obsolete for azure since all resource types there are using a class, we originally used something a little different (type string) and that was a compatibility check, we still have those classic non resource type as class for pseudo resources like an account there. nutshell whatever is cleanest is probably the best.